examples/todomvc/src/containers/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import TodoTextInput from '../components/TodoTextInput'
+import { addTodo } from '../actions'
+
+export const Header = ({ addTodo }) => (
+  <header className="header">
+    <h1>todos</h1>
+    <TodoTextInput
+      newTodo
+      onSave={(text) => {
+        if (text.length !== 0) {
+          addTodo(text)
+        }
+      }}
+      placeholder="What needs to be done?"
+    />
+  </header>
+)
+
+Header.propTypes = {
+  addTodo: PropTypes.func.isRequired
+}
+
+export default connect(null, { addTodo })(Header)
```

Comment:
Even more reasons to use the convention described above then if this is already in use here.